### ISSUE TITLE
Page transition blocked on 2.1.9

### ISSUE THREAD

**Issue by martindusinberre** [uzh/marugoto#51]
This has just occurred as I was trying to play through to test #28. 
I'm on 2.1.9 (having come via 2.1.7--it's important to know this as usually I come via 2.1.6). 
I now can't transition to 2.2.1 (which is where I will go if I've come via 2.1.7) because when I click on "Continue", nothing happens.

**Comment by martindusinberre** [comment]
I'm playing on safari, incidentally.

**Comment by lfeine** [comment]
<img width="630" alt="Screenshot 2019-09-09 at 11 58 51" src="https://user-images.githubusercontent.com/54706676/64602106-70ba1100-d3be-11e9-95f5-cc29a7f13c3c.png">

Screenshot of the console when this bug happens. (Screenshot taken on either Firefox 69.0 or Chrome 76.0.3809.132, it occured in both browsers)

**Comment by christian-bernet** [comment]
thanks. I is clear for me. I am on it. :-)

**Comment by christian-bernet** [comment]
It is working now. With no change from my side.

I think that the backend was not reachable or the importer was struggling.

**Comment by witlox** [comment]
it could be an issue with the database container, there are some configuration settings which may need changing:
- maximum number of memory mappings per process is 65530, which seems too low. it is recommended to set it to at least 256000
- it doesn't appear to have a large memory footprint
- currently on test, the logs aren't stored
- the service doesn't automatically restart containers

**Comment by interrogator** [comment]
Not relevant for production environment, because it only occurs when auto update happens. Probably do nothing...

**Comment by christian-bernet** [comment]
I had this problem today again.
It is because the importer was not rund completely.
The importer was stopped, because I pushed a new version of the backend to the test-system.
So I had to remove the hidden folder and rerun the importer and then the problem was gone.

The conclusion is: making the importer more stable as planned in https://github.com/uzh/marugoto/issues/21 will also solve this issue.
